install.sh

```diff
@@ -190,6 +190,7 @@ if [ $yn = Y ]; then
     cp $BUILDROOT/wayfire.desktop.in $BUILDROOT/wayfire.desktop
     sed -i "s@^Exec.*@Exec=$PREFIX/bin/startwayfire@g" $BUILDROOT/wayfire.desktop
     sed -i "s@^Icon.*@Icon=$PREFIX/share/wayfire/icons/wayfire.png@g" $BUILDROOT/wayfire.desktop
+    sudo mkdir -p $SESSIONS_DIR
```

Comment:
You should use $SUDO_FOR_SESSIONS instead of plain sudo here

Reply:
good catch, fixed :)